packages/framework/src/Command/CheckRabbitMqCommand.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Command;
+
+use PhpAmqpLib\Connection\AbstractConnection;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class CheckRabbitMqCommand extends Command
+{
+    protected const RETURN_CODE_OK = 0;
+    protected const RETURN_CODE_ERROR = 1;
+
+    /**
+     * @var string
+     */
+    protected static $defaultName = 'shopsys:rabbitmq:check-availability';
+
+    /**
+     * @var \PhpAmqpLib\Connection\AbstractConnection
+     */
+    protected $rabbitMqConnection;
+
+    /**
+     * @param \PhpAmqpLib\Connection\AbstractConnection $rabbitMqConnection
+     */
+    public function __construct(AbstractConnection $rabbitMqConnection)
+    {
+        parent::__construct();
+
+        $this->rabbitMqConnection = $rabbitMqConnection;
+    }
+
+    protected function configure()
+    {
+        $this->setDescription('Checks availability of Redis');
```

Comment:
should be RabbitMQ :wink:

Reply:
It is, you probably commented it in some older commit